target/linux/brcmiproc/README

```diff
@@ -0,0 +1,2 @@
+This is intended to be used with OpenWRT project to provide images for use with
+Broadcom iProc based SoCs
```

Comment:
I do not think we really need that file.

Reply:
Removed this file